<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainScreen">
   <widget class="QPushButton" name="viewSched">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>685</y>
      <width>300</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>View Schedule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manSched">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>685</y>
      <width>300</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Manage Schedule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="records">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>685</y>
      <width>300</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Records</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>1032</x>
      <y>685</y>
      <width>300</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>655</height>
     </rect>
    </property>
    <widget class="QWidget" name="Login">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>220</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/profile.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>270</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>220</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>230</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Logout">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>170</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ViewSchedule">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>210</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>View Schedule</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ManageSchedule">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>391</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Manage Schedule</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Records">
     <widget class="QPushButton" name="scope">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>230</y>
        <width>261</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Scope</string>
      </property>
     </widget>
     <widget class="QPushButton" name="washer">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>230</y>
        <width>261</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Washer</string>
      </property>
     </widget>
     <zorder>washer</zorder>
     <zorder>scope</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
